pkg/litmos/client.go

```diff
@@ -1,9 +1,12 @@
+//nolint:bodyclose // Response bodies are closed by the uhttp wrapper
```

Comment:
It doesn't hurt to close response bodies from uhttp, as they're no op readers. But it would be nice if the linter understood that they don't *need* to be closed.